.github/workflows/integration.yml

```diff
@@ -297,6 +306,9 @@ jobs:
           name: ui
           options: --mount type=bind,source=${{ github.workspace }}/ui/docker/nginx/config/local.conf,destination=/etc/nginx/conf.d/default.conf
 
+      - name: Set hosts on ui
+        run: docker exec ui sh -c 'echo -e "127.0.0.1 hep-web\n127.0.0.1 next-web\n127.0.0.1 record-editor" >> /etc/hosts'
```

Comment:
This is not the correct way to modify the hosts file.
https://docs.docker.com/engine/reference/commandline/run/#add-entries-to-container-hosts-file---add-host

Reply:
:+1: Thanks, I completely forgot about that.